app/boards/joric/nrfmicro/nrfmicro_nrf52833.dts

```diff
@@ -92,7 +97,7 @@ zephyr_udc0: &usbd {
 &flash0 {
     /*
      * For more information, see:
-     * http://docs.zephyrproject.org/latest/devices/dts/flash_partitions.html
+     * http: //docs.zephyrproject.org/latest/devices/dts/flash_partitions.html
```

Comment:
The NCS extension seems to have some interesting ideas about how things should be formatted...